app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -390,6 +391,8 @@
   <dimen name="solution_summary_margin_bottom_linear_layout">32dp</dimen>
   <dimen name="solution_summary_margin_top_solution_summary_label">16dp</dimen>
 
+  <!-- Story Fragment dimens. -->
```

Comment:
Nit
`<!-- Story Fragment dimens -->`

Also define new values at bottom.

Reply:
okk, I change it.